Source/WebCore/rendering/RenderQuote.cpp

```diff
@@ -99,28 +98,36 @@ struct SubtagComparison {
     int comparison;
 };
 
-static SubtagComparison subtagCompare(const char* key, const char* range)
+static int strcmpSpan(std::span<const LChar> s1, std::span<const LChar> s2)
```

Comment:
I might name this `cmpSpan` rather than `strcmpSpan`, since `str` is the standard library’s name for "null-terminated array of `char`".

Reply:
We already have a `equalSpans()` which wraps memcmp() for spans. However, it is only useful to compare spans of the same size. memcmp() has to specify the number of bytes we want to compare.

Here, we're really looking for a strcmp replacement which works even if the 2 strings have different sizes and will not simply compare the first N bytes. I understand though that using the term "strcmp" in the function name is a bit confusing since:
- it doesn't call strcmp() internally
- It doesn't deal with null-terminated strings.

Not quite sure how to communicate that this is basically the behavior of strcmp but with spans instead of null-terminated strings.

Reply:
I think for now I'll use this unless someone has a better proposal:
```
template<typename T>
int compareSpans(std::span<const T> a, std::span<const T> b)
{
    static_assert(std::has_unique_object_representations_v<T>);
    int result = memcmp(a.data(), b.data(), std::min(a.size(), b.size()));
    if (!result && a.size() != b.size())
        result = (a.size() > b.size()) ? 1 : -1;
    return result;
}
```

Reply:
That sounds good. I was thinking that “cmp” was the nickname for the -1/0/1 result. But I think “compare” is also fine. I should also note that the modern C++ way to do this is sort of thing is to use `std::strong_ordering` instead.

Reply:
This seems like the correct logic for byte spans of different lengths, so I don’t think there’s any need for the word “string”.